doc/builders/images/appimagetools.section.md

```diff
@@ -28,21 +28,29 @@ Note how the type 1 AppImage is described as an `ISO 9660 CD-ROM filesystem`, an
 
 ## Wrapping {#ssec-pkgs-appimageTools-wrapping}
 
-Depending on the type of AppImage you're wrapping, you'll have to use `wrapType1` or `wrapType2`.
+To create a derivation for a appimage you can use the `wrapType2` builder, which is a wrapper around `buildFHSUserEnv` that extracts the appimage for you and generates a FHS environment to run it in. Any attribute `buildFHSUserEnv` accepts will also be valid in `wrapType2` derivations.
 
 ```nix
-appimageTools.wrapType2 { # or wrapType1
-  name = "patchwork";
+appimageTools.wrapType2 rec {
+  pname = "patchwork";
+  version = "4.11.4";
+
   src = fetchurl {
-    url = "https://github.com/ssbc/patchwork/releases/download/v3.11.4/Patchwork-3.11.4-linux-x86_64.AppImage";
+    url = "https://github.com/ssbc/patchwork/releases/download/v${version}/Patchwork-${version}-linux-x86_64.AppImage";
     sha256 = "1blsprpkvm0ws9b96gb36f0rbf8f5jgmw4x6dsb1kswr4ysf591s";
   };
+
   extraPkgs = pkgs: with pkgs; [ ];
+
+  extraInstall = appimageContents: ''
+    cp -r ${appimageContents}/usr/share/icons $out/share
+  '';
 }
 ```
 
-- `name` specifies the name of the resulting image.
+- `name` specifies the name of the resulting image. Note that `pname` and `version` can be specified instead as well.
```

Comment:
I think it would help readability a lot to have the key statement in a brief sentence, and additional information in a subsequent paragraph. The manual is already very densely written, and it would probably help skimming it if it had more visual structure.

Adding it just here would of course make it inconsistent with the existing stuff. But maybe it would be good to start somewhere at all.

```suggestion
- `name` specifies the name of the resulting image.

   Note that `pname` and `version` can be specified instead as well.

```

Reply:
Good idea, I will switch over the other flags as well.